easybuild/easyconfigs/p/PlasFlow/PlasFlow-1.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,49 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'PlasFlow'
+version = '1.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/smaegol/PlasFlow'
+description = """PlasFlow - predicting plasmid sequences in metagenomic data"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['352d5d7cd73672130e32f3f61c3f2ad95aa684d7aebf7668ab3f2594996c338b']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Perl', '5.30.0'),
+    ('R', '3.6.2'),
+    ('BioPerl', '1.7.2'),
+    ('Biopython', '1.75', versionsuffix),
+    ('R-bundle-Bioconductor', '3.10'),
+    ('rpy2', '3.2.6', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('TensorFlow', '1.15.0', versionsuffix),
```

Comment:
With the merge of #9538, a violation against the "one dependency version per easyconfigs generation" rule was introduced, so I'm bumping this to TensorFlow 2.1.0 in #9947...